lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/prefer_const_border_radius.dart

```diff
@@ -15,7 +16,8 @@ part 'visitor.dart';
 
 class PreferConstBorderRadiusRule extends FlutterRule {
   static const ruleId = 'prefer-const-border-radius';
-  static const _issueMessage = 'Prefer use const constructor BorderRadius.all.';
+  static const _issueMessage =
+      'Prefer to use const constructor BorderRadius.all.';
```

Comment:
Let's have either `Prefer using const constructor BorderRadius.all` or `Prefer const constructor BorderRadius.all.`. We don't use `to use` in any other rule.